src/app/app.module.js

```diff
@@ -1,14 +1,15 @@
 //Angular and Angularmd
 import angular from 'angular';
-import angularMd from 'angular-material';
-import angularAnimate from 'angular-animate';
-import angularAria from 'angular-aria';
-import angularMessages from 'angular-messages';
+import angularmd from 'angular-material';
+import angularanimate from 'angular-animate';
+import angulararia from 'angular-aria';
+import angularmessages from 'angular-messages';
```

Comment:
These names were right, camelCase format is better

Reply:
Maybe did a bad merge from dev, sorry!